.gitmodules

```diff
@@ -1,63 +1,3 @@
 [submodule "agora"]
 	path = submodules/agora
```

Comment:
Check [this](https://git-scm.com/docs/gitmodules#Documentation/gitmodules.txt-submoduleltnamegtfetchRecurseSubmodules) for recursive submodule init